README.md

```diff
@@ -1 +1,113 @@
-# terraform-cloud-connect
\ No newline at end of file
+# terraform-cloud-connect
+
+## Description
+
+The main purpose of this script is to create an IAM User with enough permission and a Security Group with correct Inbound and Outbound rules so that Bitmovin's Cloud Connect product will be able to run Video Encodings in the configured infrastructure.
+
+The Terraform Cloud Connect module does the following:
+
+- Creates new IAM user
+- Assigns required permission required for Cloud-Connect: ~ AmazonEC2FullAccess
+- Creates a Security Group
+- Adds required Inbound and Outbound rules
+- Outputs:
+    - Account Id
+    - Access Key
+    - Secret Access Key
+    - Security Group Id
+
+The outputs that are produced by the script can be used in a next step to configure the Bitmovin Encoder for Cloud Connect.
+
+## Usage
+
+Install Terraform: 
+
+https://developer.hashicorp.com/terraform/install
+
+Run:
+
+````
+terraform init
+terraform plan
+terraform apply
+````
+
+You can use the Bitmovin Cloud Connect Terraform module like this:
+
+````
+module "bitmovin_cloud_connect" {
+  source  = "bitmovin/terraform-cloud-connect"
+
+  user_name = "bitmovin-cloud-connect-user"
+  policy_name = "bitmovin-inline-policy"
+  security_group_name = "bitmovin-security-group"
+
+  live_rtmp = false
+  live_srt = false
+  live_zixi = false
+
+  live_ingress_cidr_blocks = ["0.0.0.0/0"]
+  live_ingress_ipv6_cidr_blocks = ["::/0"]
+
+  tags = {
+           company = "bitmovin",
+           product = "cloud-connect"
+         }
```

Comment:
as commented in the other file, maybe not specifying the variables in the example but instead let the module fall back to the default looks a bit easier/cleaner

Reply:
Done